dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/action/ViewCMSMaintenanceAction.java

```diff
@@ -244,11 +277,46 @@ else if(cmd.equals("dropoldassets")){
 			boolean dataOnly = Parameter.getBooleanFromString(req.getParameter("dataOnly"), true);
 
 			if(cmd.equals("createZip")) {
+				if(!dataOnly){
+					moveAssetsToBackupDir();
+				}
+				message = "Creating XML Files. ";
+				createXMLFiles();
+				String x = UtilMethods.dateToJDBC(new Date()).replace(':', '-').replace(' ', '_');
+				File zipFile = new File(backupFilePath + "/backup_" + x + "_.zip");
+				message +="Zipping up to file:" + zipFile.getAbsolutePath();
+				final BufferedOutputStream bout = new BufferedOutputStream(Files.newOutputStream(zipFile.toPath()));
+
+				zipTempDirectoryToStream(bout);
+				message +=". Done.";
 
+//			}else if(cmd.equals("wipeOutDotCMSDatabase")) {
+//
+//				message = "Deleted dotCMS Database";

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)